responsibleai/responsibleai/_tools/causal/dashboard_schemas/schema_0.1.0.json

```diff
@@ -0,0 +1,220 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "AzureML Responsible AI causal analysis dashboard schema",
+  "description": "Full schema",
+  "type": "object",
+  "properties": {
+    "id": {
+      "description": "ID of causal result",
+      "type": "string"
+    },
+    "version": {
+      "description": "Version of causal result",
+      "type": "string"
+    },
+    "config": {
+      "type": "object",
+      "properties": {
+        "treatment_features": {
+          "description": "List of treatment features",
+          "type": "array",
+          "items": { "type": "string" }
+        }
+      },
+      "required": ["treatment_features"]
+    },
+    "global_effects": {
+      "description": "Causal effects for whole dataset",
+      "type": "array",
+      "items": { "$ref": "#/definitions/causal_effect" }
+    },
+    "local_effects": {
+      "description": "Causal effects per sample",
+      "type": "array",
+      "items": {
+        "description": "Causal effects for one sample",
+        "type": "array",
+        "items": { "$ref": "#/definitions/causal_effect" }
+      }
+    },
+    "policies": {
+      "description": "Treatment policies",
+      "type": "array",
+      "items": { "$ref": "#/definitions/policy" }
+    }
+  },
+  "required": [
+    "id",
+    "version",
+    "config",
+    "global_effects",
+    "local_effects",
+    "policies"
+  ],
+  "definitions": {
+    "causal_effect": {
+      "type": "object",
+      "properties": {
+        "sample": {
+          "description": "Number of local sample (only for local effects)",
+          "type": "number"
+        },
+        "feature": {
+          "description": "Treatment feature name",
+          "type": ["string", "number"]
+        },
+        "feature_value": {
+          "description": "Treatment feature value",
+          "type": "string"
```

Comment:
it can only be a string value?

Reply:
You're right, it can be numeric as well. Thanks for catching that